SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -290,6 +294,15 @@ void CTurbSSTSolver::Source_Residual(CGeometry *geometry, CSolver **solver_conta
   SU2_OMP_FOR_DYN(omp_chunk_size)
   for (unsigned long iPoint = 0; iPoint < nPointDomain; iPoint++) {
 
+    /*--- If fixed values apply, there is no need to compute the source term. ---*/
+    if(config->GetTurb_Fixed_Values()){
+      su2double* velocity_inf = config->GetVelocity_FreeStreamND();
+      su2double sp =
+        GeometryToolbox::DotProduct(nDim, geometry->nodes->GetCoord(iPoint), velocity_inf)
+        / GeometryToolbox::Norm(nDim, velocity_inf);
```

Comment:
Is it worth checking the velocity is not zero somewhere?

Reply:
Yes, I'll check and also normalize the velocity vector outside the loop.